content/es/docs/concepts/configuration/secret.md

```diff
@@ -0,0 +1,907 @@
+---
+reviewers:
+- mikedanese
```

Comment:
nit: nominar a un reviewer que hable español

Reply:
@raelga, podría apoyarme asignando a un Reviewer que hable espeañol? , por favor :)

Reply:
Alternative: remove the `reviewers` key?

Reply:
```suggestion
- raelga
```

Si te parece bien @editaxz, me asigno yo mismo como reviewer.